kolibri/content/recommendations.py

```diff
@@ -0,0 +1,32 @@
+from collections import Counter
+
+# from kolibri.logger.models import ContentInteractionLog
+from kolibri.content import api
+from kolibri.content.models import ContentNode
+
+def recommendations_interaction_log():
+    """
+    Calculates the content that is accessed the most and returns a subset of that.
+
+    :return: Queryset of the the most frequently accessed content
+    """
+    # logs = ContentInteractionLog.objects.filter(channel_id=channel_id).all()
+    logs = ContentNode.objects.all()
+    content_count = Counter()
+    for log in logs:
+        content_count[log.content_id] += 1
```

Comment:
See here for an efficient way to do this: https://docs.djangoproject.com/en/1.9/topics/db/aggregation/#cheat-sheet